tests/serialize/data.js

```diff
@@ -99,7 +99,7 @@ var doNext = function (remaining) {
             return
           }
           doc = $rdf.sym($rdf.uri.join(right, base))
-          if (doc.uri.slice(0, 8) !== 'file:///') {
+          if (doc.uri.slice(0, 7) !== 'file://') {
```

Comment:
So the problem with doc.uri.slice(0, 8) !== 'file:///' is that on windows process.cwd() doesn't place the starting / on the folder name, which actually turns out to be ok since then you slice "file//" away and fs.writeFile automatically works on both systems. I know the protocol require file:/// but that code actually only lives on those tests, and there's no entry point for a full URI that might be parsed wrong. I was trying very hard to not add any if(onWindows) statements since fs and xmlhttprequest handle the "slash promiscuity" pretty well but I can formalize it more if you want. I've been working on refactoring that whole testing batch with mocha/chai and promises anyway.